bridge_adaptivity/bridge_lti/models.py

```diff
@@ -32,19 +31,19 @@ def __str__(self):
 
 
 @python_2_unicode_compatible
-class LtiConsumer(ModelFieldIsDefaultMixin, models.Model):
+class LtiConsumer(models.Model):
```

Comment:
We remove IsDefault mixin, what about the case with no active source, I would add the new mixing with the check there is at least one active source and point admin if he trying to switch off all sources.

Reply:
make sense